### DISCUSSION TITLE
[Improvement]: Add Synology/Container Manager support

### DISCUSSION THREAD

**Discussion by moshechaikin** [coollabsio/coolify#3166] [Improvement Requests]
### Description

Installing Coolify does not work with Synology (Docker Container Manager)

### Minimal Reproduction (if possible, example repository)

When running Coolify install script (as root) via SSH on a Synology device, this occurs:
`grep: /etc/os-release: No such file or directory`

Happy to test fixes on my device
### Exception or Error

`grep: /etc/os-release: No such file or directory`

### Version

latest

**Comment by esausilva** [comment]
Looks like you can install a VM to run Coolify https://x.com/wesbos/status/1788590332981944667 

Not ideal, but it looks like it works https://x.com/wesbos/status/1788618933576106174

**Comment by moshechaikin** [comment]
Interesting, thanks. I guess better than nothing.

**Comment by savanvyas** [comment]
this would great to have

**Comment by mihneawalker** [comment]
I am currently trying to install coolify on a Synology nas. I installed Portainer, and from there, I am trying to make it work, but I am not familiar with some concepts, like how should I present the env vars for example. So far I haven't had too much luck, maybe someone here has figured it out...

**Comment by esausilva** [comment]
@mihneawalker Maybe this would be useful for you https://www.portainer.io/blog/using-env-files-in-stacks-with-portainer

**Comment by andrzejSulkowski** [comment]
Any updates? Have you figured it out?

**Reply by moshechaikin** [reply]
Running in a Ubuntu-server VM
